lib/channel-websocket/src/index.js

```diff
@@ -52,6 +52,8 @@ export class WebsocketTransport {
       this._handler(event);
     };
     this._socket.onerror = e => {
+      // Ignore network errors like `ECONNRESET`, `EPIPE`, etc.
+      if (e.errno) return;
```

Comment:
This will also catch connection errors like `ECONNREFUSED`, not sure if wanted.

Reply:
What would be the proper filter to only catch crome tab reload?

Reply:
Is `WebSocket` here the browser native `WebSocket` object or `ws` or it depends on the context?

Reply:
I think this is the browser `WebSocket` as per https://github.com/storybooks/storybook/pull/3002/files#diff-432fa23ac718b94ed5be1b2802371d41R3. In this case you don't need this, the fix on the server is sufficient and correct.